2021/1002xxx/GSD-2021-1002353.json

```diff
@@ -1,106 +1,6 @@
-        "data_type": "CVE",
-        "data_version": "4.0",
-        "impact": {
-          "cvss": {
-            "attackVector": "NETWORK",
-            "attackComplexity": "LOW",
-            "privilegesRequired": "NONE",
-            "userInteraction": "NONE",
-            "scope": "CHANGED",
-            "confidentialityImpact": "HIGH",
-            "integrityImpact": "HIGH",
-            "availabilityImpact": "HIGH",
-            "baseScore": 10,
-            "baseSeverity": "CRITICAL",
-            "exploitCodeMaturity": "HIGH",
-            "remediationLevel ": "OFFICIALFIX",
-            "reportConfidence": "CONFIRMED",
-            "temporalScore": 9.5,
-            "temporalSeverity": "CRITICAL",
-            "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:C/C:H/I:H/A:H/E:H/RL:O/RC:C",
-            "version": "3.1"
-          }
-        },
-        "affects": {
-          "vendor": {
-            "vendor_data": [
-              {
-                "vendor_name": "Red Hat",
-                "product": {
-                  "product_data": [
-                    {
-                      "product_name": "OpenShift",
-                      "version": {
-                        "version_data": [
-                          {
-                            "version_affected": "=",
-                            "version_name": "OpenShift Container Platform 4",
-                            "version_value": "4"
-                          }
-                        ]
-                      }
-                    }
-                  ]
-                }
-              }
-            ]
-          }
-        },
-        "description": {
-          "description_data": [
-            {
-              "lang": "eng",
-              "value": "The vulnerability reported in GSD-2021-1002352 (CVE-2021-44228) was not correctly fixed \"in certain non-default configuration\" and a new vulnerability and patch have been released, as such Apache log4js 2.15 is vulnerable. This could allows attackers with control over Thread Context Map (MDC) input data when the logging configuration uses a non-default Pattern Layout with either a Context Lookup (for example, $${ctx:loginId}) or a Thread Context Map pattern (%X, %mdc, or %MDC) to craft malicious input data using a JNDI Lookup pattern resulting in a denial of service (DOS) attack. Log4j 2.15.0 restricts JNDI LDAP lookups to localhost by default. Note that previous mitigations involving configuration such as to set the system property `log4j2.noFormatMsgLookup` to `true` do NOT mitigate this specific vulnerability. Log4j 2.16.0 fixes this issue by removing support for message lookup patterns and disabling JNDI functionality by default. This issue can be mitigated in prior releases (<2.16.0) by removing the JndiLookup class from the classpath (example: zip -q -d log4j-core-*.jar org/apache/logging/log4j/core/lookup/JndiLookup.class)."
-            }
-          ]
-        }
-      }
-    }
+    "id": "GSD-2021-45046",
```

Comment:
This file isn't GSD-2021-45046...

Reply:
Yes it is, it's a duplicate. So that file is removed, go to GSD-2021-45046 for the info. It's how the other duplicates were done.